src/components/RadioButton/RadioButtonGroup.js

```diff
@@ -0,0 +1,49 @@
+import React, { forwardRef } from 'react';
+import PropTypes from 'prop-types';
+import bem from 'bem';
+import styles from './RadioButtonGroup.scss';
+
+const { block, elem } = bem({
+    name: 'RadioButtonGroup',
+    classnames: styles,
+});
+
+const RadioButtonGroup = forwardRef((props, ref) => {
+    const { children, name, onChange, ...rest } = props;
+
+    const childProps = { name };
+    if (onChange) {
+        childProps.onChange = onChange;
+    }
+
+    return (
+        <div ref={ref} {...rest} {...block(props)}>
+            {React.Children.map(children, child => {
```

Comment:
I just wondering why we need here {...block(props)} if the component doesn't have styles for block

Reply:
I think this is just the standard of how we use OneUI. It will also ensure that className for consuming application will be passed correctly.